### DISCUSSION TITLE
How to you can access Select of Antd with tags mode

### DISCUSSION THREAD

**Discussion by hieusmiths** [react-hook-form/react-hook-form#1716] [General]
Hi, lee.
To day i must add new post with keyword along.
I used <Select name='test' mode="tags"> with Controller.
Haha, i thinked your lib will not access and get the value, but no.

You can explain for me, How to you access with this component.
Thank so much.
Have a good day.

**Comment by bluebill1049** [comment]
hey @hekkim u mean to use `watch` API to read the input value?
